powersoftau/src/bin/beacon_constrained.rs

```diff
@@ -40,8 +40,8 @@ fn main() {
         use crypto::sha2::Sha256;
         use crypto::digest::Digest;
 
-        // Place block hash here (block number #602168)
-        let mut cur_hash: [u8; 32] = hex!("00000000000000000013a0dab9d26be0353108f6eb5a2be6ac389986296607c7");
+        // Place block hash here (block number #657446)
+        let mut cur_hash: [u8; 32] = hex!("0000000000000000000f361c481f11232f657054a51c4c99f9808b7c53d1e498");
```

Comment:
The block we use here we need to publicly announce before the block is mined, like we did here: https://twitter.com/loopringorg/status/1190701331771723778

Or we could just use the phase 1 files used by tornado cache, if those are available somewhere.

Reply:
OK